examples/tts/conf/audio_codec/audio_codec_24000.yaml

```diff
@@ -162,9 +162,16 @@ exp_manager:
   exp_dir: null
   name: ${name}
   create_tensorboard_logger: true
-  create_checkpoint_callback: true
-  create_wandb_logger: false
+  create_wandb_logger: true
+  wandb_logger_kwargs:
+    name: ${name}
+    project: ???
```

Comment:
lets keep this null (optional) since its not mandatory if users would like to use tensorboard

Reply:
I mean, `create_tensorboard_logger=false` and keep only wandb as default in yaml files.

Reply:
currently both tensorboard and wandb both are enabled

Reply:
yeah lets only keep wandb for now